pkgs/by-name/ad/adbtuifm/package.nix

```diff
@@ -0,0 +1,29 @@
+{
+  buildGoModule,
+  fetchFromGitHub,
+  android-tools,
+  lib,
+}:
+buildGoModule rec {
+  pname = "adbtuifm";
+  version = "0.5.8";
+  src = fetchFromGitHub {
+    owner = "darkhz";
+    repo = "adbtuifm";
+    rev = "v${version}";
+    hash = "sha256-TK93O9XwMrsrQT3EG0969HYMtYkK0a4PzG9FSTqHxAY=";
+  };
+  vendorHash = "sha256-voVoowjM90OGWXF4REEevO8XEzT7azRYiDay4bnGBks=";
+  buildInputs = [
+    android-tools
+  ];
+  meta = with lib; {
+    description = "adbtuifm is a TUI-based file manager for the Android Debug Bridge, to make transfers between the device and client easier";
```

Comment:
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes

> Not start with the package name.
> 
>     More generally, it should not refer to the package name.
> 

> Be short, just one sentence.

Reply:
Okay, I can fix that. 

As a general question, for small changes like this for a PR on nixpkgs, would it be better to have multiple commits fixing different issues or rather to rebase every time something needs to be changed?

Reply:
https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#commit-conventions

one commit per "unit"

Reply:
I see, thank you